pennylane/labs/resource_estimation/__init__.py

```diff
@@ -50,9 +50,11 @@
 .. autosummary::
     :toctree: api
 
+    ~ResourceAdjoint
```

Comment:
Can you add a separate section titled `Symbolic Operators` and place the 3 symbolic operators there.